docs/management/console-rack-management.md

```diff
@@ -59,3 +59,65 @@ You can move any Console-managed Rack back to being locally managed only with th
     staging            gcp       running
 ```
 Terraform state will be transferred to your local machine for exclusive management.
+## View Kubernetes resources
+You can view the Kubernetes resources deployed to your cluster with the AWS Management Console.  
+
+### Steps:
+- Create a Kubernetes clusterrolebinding that is bound to a Kubernetes clusterrole that has the necessary permissions to view the Kubernetes resources. To learn more about Kubernetes roles and role bindings, see Using RBAC Authorization in the Kubernetes documentation. You can apply one of the following manifests to your cluster that create a role and rolebinding or a clusterrole and clusterrolebinding with the necessary Kubernetes permissions: *
+If your rack version is > 3.6.4, you don’t need to create the Cluster role and the Cluster role binding.
+```
+kubectl apply -f https://s3.us-west-2.amazonaws.com/amazon-eks/docs/eks-console-full-access.yaml
+```
+- Make sure that the eks:AccessKubernetesApi, and other necessary IAM permissions to view Kubernetes resources, are assigned to either the user that you sign into the AWS Management Console with, or the role that you switch to once you've signed in to the console.
+The following example policy includes the necessary permissions for a user or role to view Kubernetes resources for all clusters in your account. Replace `111122223333` with your account ID.
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "eks:ListFargateProfiles",
+                "eks:DescribeNodegroup",
+                "eks:ListNodegroups",
+                "eks:ListUpdates",
+                "eks:AccessKubernetesApi",
+                "eks:ListAddons",
+                "eks:DescribeCluster",
+                "eks:DescribeAddonVersions",
+                "eks:ListClusters",
+                "eks:ListIdentityProviderConfigs",
+                "iam:ListRoles"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "ssm:GetParameter",
+            "Resource": "arn:aws:ssm:*:111122223333:parameter/*"
+        }
+    ]
+}   
+```
+- For more information about adding users or roles to the aws-auth ConfigMap, see [Add IAM users, roles, or AWS accounts to the ConfigMap](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html#aws-auth-users).
```

Comment:
Can you add a warning here that misconfiguring the `aws-auth` can lock the user out of their rack and they must proceed with caution?

Reply:
sure!